dolphin/drivers/dell_emc/vmax/client.py

```diff
@@ -123,3 +123,58 @@ def list_pools(self):
             LOG.error("Failed to get pool metrics from VMAX: {}".format(err))
             raise exception.StorageBackendException(
                 reason='Failed to get pool metrics from VMAX')
+
+    def list_volumes(self, storage_id):
+
+        try:
+            # List all volumes
+            volumes = self.conn.provisioning.get_volume_list(filters={'data_volume': 'false'})
```

Comment:
Will this raise any exception? I don't find it in PyU4V code but can u please confirm

Reply:
Yes. PyU4V uses requests module. https://requests.readthedocs.io/en/latest/user/quickstart/#errors-and-exceptions

Reply:
Ok. My main concern was to catch any exception and propagate it to upper layer too. If client wants to do RCA based upon these, it will help.

Reply:
Opened an issue to track